enterprise/internal/batches/resolvers/credential_test.go

```diff
@@ -60,3 +63,13 @@ func TestUnmarshalBatchChangesCredentialID(t *testing.T) {
 		}
 	}
 }
+
+func TestCommentSSHKey(t *testing.T) {
+	publicKey := "public"
```

Comment:
```suggestion
	publicKey := "public\n"
```

To make sure your `TrimRight` is doing something :)